src/assets/i18n/en.json

```diff
@@ -1,44 +1,98 @@
   "BY_RATING": "By rating",
   "CANCEL": "Cancel",
   "CHANGE": "Change",
+  "CHANGE_EMAIL": "Change email",
+  "CHANGE_PASSWORD": "Change password",
   "CLEAR_FILTERS": "Clear Filters",
   "CLOSE_RECRUITMENT": "Closed Recruitment",
   "CONFIRM": "Confirm",
   "CONTACTS": "CONTACTS",
+  "DATE_OF_BIRTH_OF_THE_MANAGER": "Date of birth of the manager",
   "DAYS": {
-    "MONDAY": "Mon",
-    "TUESDAY": "Tue",
-    "WEDNESDAY": "Wed",
-    "THURSDAY": "Thu",
     "FRIDAY": "Fri",
+    "MONDAY": "Mon",
     "SATURDAY": "Sat",
-    "SUNDAY": "Sun"
+    "SUNDAY": "Sun",
+    "THURSDAY": "Thu",
+    "TUESDAY": "Tue",
+    "WEDNESDAY": "Wed"
   },
   "DIRECTIONS": "Categories",
+  "DIRECTOR_NAME": "Director's full name",
   "DISPLAY_ON_THE_PAGE": "Items per page:",
+  "DISTRICT_OF_THE_CITY": "District of the city",
   "DO_YOU_PROVIDE_EDUCATIONAL_SERVICES": "Do you provide educational services?",
+  "EDIT": "Edit",
+  "EDPNOU": "EDPNOU",
   "ENTER_THE_NAME_OF_THE_GROUP_SCHOOL": "Enter the name of the group, school",
   "ENUM": {
     "CHILD_DECLINATION": {
       "CHILD": "child",
       "CHILD_GENITIVE": "children",
+      "CHILDREN": "children",
       "CHILDREN_ABLATIVE": "children",
-      "NO_CHILDREN": "No children",
-      "CHILDREN": "children"
+      "NO_CHILDREN": "No children"
+    },
+    "NAV_BAR_NAME": {
+      "ABOUT_PORTAL": "About portal",
+      "ADD_ACHIEVEMENT": "Add achievement",
+      "ADD_ADMIN": "Add admin",
+      "ADD_DEPUTY_DIRECTOR": "Add deputy director",
+      "ADD_DIRECTION": "Add direction",
+      "ADD_INFORMATION_ABOUT_CHILD": "Add information about child",
+      "ADD_WORKSHOP_ADMIN": "Add workshop admin",
+      "ADMINISTRATION": "Administration",
+      "ADMINS": "Admins",
+      "APPLICATIONS": "Applications",
+      "CHAT": "Chat",
+      "DIRECTIONS": "Directions",
+      "EDIT_ACHIEVEMENT": "Edit achievement",
+      "EDIT_ADMIN": "Edit admin",
+      "EDIT_DIRECTION": "Edit direction",
+      "EDIT_INFORMATION_ABOUT_CHILD": "Edit information about child",
+      "EDIT_INSTITUTION": "Edit institution",
+      "EDIT_PERSONAL_INFORMATION": "Edit personal information",
+      "EDIT_WORKSHOP": "Edit workshop",
+      "FAVORITE": "Favorite",
+      "HISTORY_OF_CHANGES": "History of changes",
+      "INFORMATION_ABOUT_CHILD": "Information about child",
+      "INFORMATION_ABOUT_CHILDREN": "Information about children",
+      "INFORMATION_ABOUT_PROVIDER": "Information about provider",
+      "MAIN": "Main",
+      "MAIN_PAGE": "Main page",
+      "MESSAGES": "Messages",
+      "NEW_WORKSHOP": "New workshop",
+      "NORMATIVE_LEGAL_ACTS": "Normative legal acts",
+      "PERSONAL_INFORMATION": "Personal information",
+      "PLATFORM": "Platform",
+      "PORTAL": "Portal",
+      "PROVIDERS": "Providers",
+      "REQUEST_ON_WORKSHOP": "Request on workshop",
+      "SEARCH_RESULT": "Search result",
+      "SUPPORT": "Support",
+      "TOP_DESTINATION": "Top destination",
+      "UPDATE_ADMIN": "Update admin",
```

Comment:
do not use shorts, for translation it has to be the whole word -> administrator
don't use "update", stick with the one translation variant, in our case  EDIT